CenterForOpenScience/ember-osf-reviews
# [IN-168][Reviews] Add Reviews Notifications page

## Purpose

This PR adds Notifications page to Reviews app.

## Summary of Changes

1. Added new component `notification-list`.
2. Added new component `notification-list-row`.
3. Changed `reviews-breadcrumbs/component.js` to capitalize the breadcrumb.
4. Added tests for new components and routes.

## Side Effects / Testing Notes

None.

## Testing Notes

This PR, once merged, plays with the previously added `/subscriptions` endpoint (https://openscience.atlassian.net/browse/IN-167) so that users are able to change their subscription.
Several things to test:
1. Make sure that changing the notification frequency actually works.
2. Make sure there is placeholder during initial loading of the page. Placeholder is not needed when you change the notification frequency.
3. Make sure that alerts are shown when updating notification fails.
4. Make sure that it looks good.

## Ticket

https://openscience.atlassian.net/browse/IN-168
